pkgs/development/compilers/llvm/git/clang/gnu-install-dirs.patch

```diff
@@ -1,71 +1,137 @@
-diff --git a/cmake/modules/AddClang.cmake b/cmake/modules/AddClang.cmake
-index 75b0080f6715..c895b884cd27 100644
---- a/cmake/modules/AddClang.cmake
-+++ b/cmake/modules/AddClang.cmake
-@@ -119,8 +119,8 @@ macro(add_clang_library name)
-         install(TARGETS ${lib}
-           COMPONENT ${lib}
-           ${export_to_clangtargets}
--          LIBRARY DESTINATION lib${LLVM_LIBDIR_SUFFIX}
--          ARCHIVE DESTINATION lib${LLVM_LIBDIR_SUFFIX}
-+          LIBRARY DESTINATION "${CMAKE_INSTALL_LIBDIR}${LLVM_LIBDIR_SUFFIX}"
-+          ARCHIVE DESTINATION "${CMAKE_INSTALL_LIBDIR}${LLVM_LIBDIR_SUFFIX}"
-           RUNTIME DESTINATION "${CMAKE_INSTALL_BINDIR}")
+diff --git a/CMakeLists.txt b/CMakeLists.txt
```

Comment:
I wonder why the patch has changed...

Reply:
Idk either, could be things got mixed up? Should I just copy what we got in master?

Reply:
Yes. This file looks like `pkgs/development/compilers/llvm/git/llvm/gnu-install-dirs.patch`, actually.

Reply:
Yeah, I just copied from master and stable right now for this patch but diff says they're the same file. GitHub seems to think otherwise.

Reply:
Probably `git checkout HEAD^ pkgs/development/compilers/llvm/git/clang/gnu-install-dirs.patch` could help?

Reply:
Well, I'll have to try that this evening.

Reply:
> `Updated 0 paths from 90acbde62641`

I guess that means it's good?

Reply:
Okay, fixed